validators/smoke/genesis-chain/test.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Simply query the 5th block and check that it's correct
-reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":0}' "$HIVE_CLIENT_IP:8545"`
+reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":1}' "$HIVE_CLIENT_IP:8545"`
```

Comment:
It turns out that's a known issue and is due to a bug in tinyrpc: https://github.com/ethereum/pyethapp/issues/181

tinyrpc doesn't seem to be actively maintained anymore, so I won't even bother trying to get it fixed there. I suppose, in the long term, what we need is to replace tinyrpc with something else in pyethapp, but in the meantime should we just not bother with having pyethapp passing the smoke tests or would it be ok to use id=1 in those tests, with a comment explaining why we can't use id=0?